wicket-core/src/test/java/org/apache/wicket/protocol/http/ClientPropertiesTest.java

```diff
@@ -149,4 +149,32 @@ void timezoneAET()
 
 		assertEquals(TimeZone.getTimeZone("AET"), props.getTimeZone());
 	}
+
+	/**
+	 * jsTimeZone "positive" test
+	 */
+	@Test
+	void timezoneJsPositive()
+	{
+		ClientProperties props = new ClientProperties();
+		props.setUtcOffset("11");
+		props.setUtcDSTOffset("10");
+		props.setJsTimeZone("Asia/Novosibirsk");
+
+		assertEquals(TimeZone.getTimeZone("Asia/Novosibirsk"), props.getTimeZone());
+	}
+
+	/**
+	 * jsTimeZone "negative" test
+	 */
+	@Test
+	void timezoneJsNegative()
+	{
+		ClientProperties props = new ClientProperties();
+		props.setUtcOffset("11");
+		props.setUtcDSTOffset("10");
+		props.setJsTimeZone("aaa");
+
+		assertEquals(TimeZone.getTimeZone("AET"), props.getTimeZone());
```

Comment:
What is `"AET"` ?

Reply:
Please add a test with `props.getJsTimeZone()` returning `null`.

Reply:
`AET` === `Australian Eastern Standard Time` taken from the previous test
This is the time zone determined for offset specified

Reply:
@martin-g actually all `timezone*` tests in this class do perform tests with `jsTimeZone == null`
Do I really have to add one more test?

Reply:
No need then!